lonestar/analytics/cpu/pagerank/PageRank-pull.cpp

```diff
@@ -36,6 +36,13 @@ static cll::opt<Algo> algo("algo", cll::desc("Choose an algorithm:"),
                                        clEnumVal(Residual, "Residual")),
                            cll::init(Residual));
 
+//! Flag that forces user to be aware that they should be passing in a
+//! transposed graph.
+static cll::opt<bool> transposedGraph(
+    "transposedGraph",
```

Comment:
This could be moved to the main LonestarStart.h header, but it's probably not used enough in other apps to need that since not many applications explicitly take a transpose graph.

It's fine as is.